libraries/AC_AttitudeControl/AC_PosControl.h

```diff
@@ -272,6 +272,12 @@ class AC_PosControl
     /// get desired roll, pitch which should be fed into stabilize controllers
     float get_roll() const { return _roll_target; }
     float get_pitch() const { return _pitch_target; }
+    float get_heading() const { return _heading; }
```

Comment:
I think we should add some comments above get_heading() and get_rate_heading() to clarify the different.  Maybe these are the same as the AC_WPNav::get_yaw() and get_yaw_rate_cds() so they could be named similarly?